desktop/libs/aws/src/aws/tests.py

```diff
@@ -131,4 +132,25 @@ def test_with_idbroker_on_ec2(self):
     finally:
       finish()
       clear_cache()
-      conf.clear_cache()
\ No newline at end of file
+      conf.clear_cache()
+
+  def test_with_raz_enabled(self):
+    with patch('aws.client.RazS3Connection') as raz_s3_connection:
+      resets = [
+        RAZ.IS_ENABLED.set_for_testing(True),
+        conf.AWS_ACCOUNTS.set_for_testing({'default': {
+          'region': 'us-west-2',
+          'host': 's3-us-west-2.amazonaws.com',
+          'allow_environment_credentials': 'false'
+        }})
+      ]
+
+      try:
+        client = get_client(name='default', fs='s3a', user='hue')
+        assert_true(client)
+        raz_s3_connection.assert_called_with(host='s3-us-west-2.amazonaws.com', username='hue')
```

Comment:
Here would feel more proper to unit tests those two (in a conf_tests.py) as unsure if above tests what we want (and way more explicit):

```
is_enabled()
has_s3_access()
```

Reply:
@romainr  Removed the extra test check added the tests for `is_enabled()` and `has_s3_access()` in separate `conf_tests.py`.